devito/operator.py

```diff
@@ -96,10 +95,10 @@ def __init__(self, expressions, **kwargs):
         nodes = iet_build(clusters, self.dtype)
 
         # Introduce C-level profiling infrastructure
-        nodes, self.profiler = self._profile_sections(nodes, parameters)
+        nodes, self.profiler = self._profile_sections(nodes)
 
         # Translate into backend-specific representation (e.g., GPU, Yask)
-        nodes = self._specialize(nodes, parameters)
+        nodes = self._specialize(nodes)
```

Comment:
I'm pretty sure that the YASK backend modifies ``parameters``  in place, but I cannot find any changes to the YASK backend?